packages/hint-babel-config/src/is-valid.ts

```diff
@@ -41,8 +49,8 @@ export default class BabelConfigIsValidHint implements IHint {
         };
 
         context.on('parse::error::babel-config::json', invalidJSONFile);
-        context.on('parse::error::babel-config::circular', invalidJSONFile);
-        context.on('parse::error::babel-config::extends', invalidJSONFile);
+        context.on('parse::error::babel-config::circular', invalidExtends);
```

Comment:
Maybe we should merge the `parser::error::NAME::circular/extends` into a single one (`extends`?)

Reply:
SGTM